products/fairphone.md

```diff
@@ -47,7 +54,7 @@ releases:
     link: https://support.fairphone.com/hc/articles/360019515018
 
 -   releaseCycle: "1"
-    highestAndroidVersion: "4.2"
+    supportedAndroidVersions: 4.2 # https://support.fairphone.com/hc/en-us/articles/6217522827281-Fairphone-1-FAQ
```

Comment:
```suggestion
    supportedAndroidVersions: "4.2" # https://support.fairphone.com/hc/en-us/articles/6217522827281-Fairphone-1-FAQ
```

Let's use quotes here so that the version is not interpreted as a number.